assets/quantity-popover.js

```diff
@@ -37,6 +39,11 @@ if (!customElements.get('quantity-popover')) {
 
       togglePopover(event) {
         event.preventDefault();
+        if (event.type === 'mouseenter') {
```

Comment:
Small suggestion: Maybe we can convert these to variables

    ```
this.actions = {
      mouseEnter: 'mouseenter',
      click: 'click'
    }
```


And refer to it as `this.actions.click` and `this.actions.mouseEnter` throughout the file

Reply:
I don't see we use it a lot in the file, we use it only in one function. Am I getting you right that you suggest we do it this way? I'm not sure if it's worth it but I'm okay if everyone agrees. 

```suggestion
       this.actions = {
          mouseEnter: 'mouseenter',
          click: 'click'
        }

        if (event.type === this.actions.mouseEnter) {
          this.eventMouseEnterHappened = true;
        }

        if (event.type === this.actions.click && this.eventMouseEnterHappened) return;

        const button = this.infoButtonDesktop && this.mql.matches ? this.infoButtonDesktop : this.infoButtonMobile;
        const isExpanded = button.getAttribute('aria-expanded') === 'true';

        button.setAttribute('aria-expanded', !isExpanded);

        this.popoverInfo.toggleAttribute('hidden');

        const isOpen = button.getAttribute('aria-expanded') === 'true';

        button.classList.toggle('quantity-popover__info-button--open');

        if (isOpen && event.type !== this.actions.mouseEnter) {
          button.focus();
        }
```